src/StdChains.sol

```diff
@@ -118,18 +121,29 @@ abstract contract StdChains {
         idToAlias[chain.chainId] = chainAlias;
     }
 
+    function _toUpper(string memory str) private pure returns (string memory) {
+        bytes memory strb = bytes(str);
+        bytes memory copy = new bytes(strb.length);
+        for (uint256 i = 0; i < strb.length; i++) {
+            bytes1 b = strb[i];
+            if (b >= 0x61 && b <= 0x7A) {
+                copy[i] = bytes1(uint8(b) - 32);
+            } else {
+                copy[i] = b;
+            }
+        }
+        return string(copy);
+    }
+
     // lookup rpcUrl, in descending order of priority:
-    // current -> config (foundry.toml) -> default
-    function getChainWithUpdatedRpcUrl(string memory chainAlias, Chain memory chain)
-        private
-        view
-        returns (Chain memory)
-    {
+    // current -> config (foundry.toml) -> environment variable -> default
+    function getChainWithUpdatedRpcUrl(string memory chainAlias, Chain memory chain) private returns (Chain memory) {
         if (bytes(chain.rpcUrl).length == 0) {
             try vm.rpcUrl(chainAlias) returns (string memory configRpcUrl) {
                 chain.rpcUrl = configRpcUrl;
             } catch (bytes memory err) {
-                chain.rpcUrl = defaultRpcUrls[chainAlias];
+                chain.rpcUrl =
+                    vm.envOr(string(abi.encodePacked("RPC_URL_", _toUpper(chainAlias))), defaultRpcUrls[chainAlias]);
```

Comment:
Thoughts on `ALIAS_RPC_URL` instead of `RPC_URL_ALIAS`? I'm personally ok with either, but from a quick search it seems the former is a much more common naming scheme—1.1k results for the [former](https://sourcegraph.com/search?q=context:global+MAINNET_RPC_URL+count:5000&patternType=standard&sm=1), 101 for the [latter](https://sourcegraph.com/search?q=context:global+RPC_URL_MAINNET+count:5000&patternType=standard&sm=1)

![image](https://user-images.githubusercontent.com/17163988/211109862-059885dd-f44d-460b-818a-dac3513ade5e.png)
<img width="664" alt="image" src="https://user-images.githubusercontent.com/17163988/211109854-9259f1a0-bce1-4d84-8563-e6d81975e42e.png">

Reply:
I'm okay with switching the order. I suppose putting the alias in front matches the style of `ETH_RPC_URL` that dapp tools defined.

Reply:
done https://github.com/foundry-rs/forge-std/pull/268/commits/758fd9066dca595a71440c7942e8164d4b4c473c